### PR TITLE
Branch/shaikram abdulguro

### PR DESCRIPTION
**Pull request by shaikram**

### CODE REVIEW

File: shaikram_abdulguro/activity1/index.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css">
```
Comment (mrmartincarlito) [inline shaikram_abdulguro/activity1/index.html:6]: I dont usually recommend calling ng entire cdn file via online path, you can download it saka mo iimport

File: shaikram_abdulguro/activity1/index.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    <title>Log In</title>
+</head>
+<style>
+    body{
```
Comment (mrmartincarlito) [inline shaikram_abdulguro/activity1/index.html:10]: Also I don't recommend direct styling,
Pwede ka magcreate separate css and then import it

File: shaikram_abdulguro/activity1/index.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        background: #2d2f30;
+    }
+    h1{
```
Comment (mrmartincarlito) [inline shaikram_abdulguro/activity1/index.html:13]: also best practice is formatting

h1 {
<style here>
 }

Comment (mrmartincarlito) [reply shaikram_abdulguro/activity1/index.html:13]: Same with the others need ng space whenever opening a curly brace

File: shaikram_abdulguro/activity1/index.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    <center>
+        
+        <div class="container ">
+            <form onsubmit="validate();">
+                <div class="mb-3">
+                    <qh1>Log-In Page</qh1>
+                </div>
+                <div class="mb-3">
+                    <h3>Login using given data below:</h3>
+                    <span>Email: <b>kodego@yahoo.com</b></span><br><br><br>
```
Comment (mrmartincarlito) [inline shaikram_abdulguro/activity1/index.html:42]: Too much breaks,
You can use margin-bottom or padding-bottom
Or you can use table tag

File: shaikram_abdulguro/activity1/index.html [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+        float: left;
+    }
+</style>
+<body>
+    <center>
+        
+        <div class="container ">
+            <form onsubmit="validate();">
+                <div class="mb-3">
+                    <qh1>Log-In Page</qh1>
```
Comment (mrmartincarlito) [inline shaikram_abdulguro/activity1/index.html:38]: What's qh1? sometimes naming your own tag can be considered for as long as it is needed, but right now h1 can be use as is

File: shaikram_abdulguro/activity1/index.html [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+        background:#fff;
+        padding: 2em;
+        border-radius: 3px;
+    }
+    h3, span, label{
+        float: left;
+    }
+</style>
+<body>
+    <center>
```
Comment (mrmartincarlito) [inline shaikram_abdulguro/activity1/index.html:33]: center tag is deprecated already,
Bootstrap has already built in center which is align-self-center, text-center, justify-content-center

File: shaikram_abdulguro/activity1/index.html [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+                  <input type="password" class="form-control" id="exampleInputPassword1" minlength="8" maxlength="25" required>
+                </div>
+                
+                <button type="submit" class="btn btn-primary">Submit</button>
+              </form>
+        </div>
+    </center>
+    
+</body>
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/js/bootstrap.min.js" integrity="sha384-ODmDIVzN+pFdexxHEHFBQH3/9/vQ9uori45z4JjnFsRydbmQbmL5t1tQ0culUzyK" crossorigin="anonymous"></script>
```
Comment (mrmartincarlito) [inline shaikram_abdulguro/activity1/index.html:61]: Same with this, don't call it via online path, download the dependencies so you dont need integrity and crossorigin

File: shaikram_abdulguro/activity1/index.html [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+<script>
+    function validate(){
+        var email = document.getElementById('exampleInputEmail1');
```
Comment (mrmartincarlito) [inline shaikram_abdulguro/activity1/index.html:64]: ES standard is using let as a declaration type

File: shaikram_abdulguro/activity1/index.html [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+        var password = document.getElementById('exampleInputPassword1');
+        var loginData = ["kodego@yahoo.com", "webdevelopment"];
+
+        if(email.value == loginData[0] && password.value == loginData[1]){
```
Comment (mrmartincarlito) [inline shaikram_abdulguro/activity1/index.html:68]: spacing , formatting
if ( condition && condition ) {

}

also if you are conditioning a strict values use ===

**Review by mrmartincarlito — CHANGES_REQUESTED** [review]
Approved ko naman yung activity2 mo
Pero I have some requested changes for you :)

File: shaikram_abdulguro/activity1/index.html (hunk 1 above)
Comment (shaikram) [reply shaikram_abdulguro/activity1/index.html:6]: ok sir, noted

File: shaikram_abdulguro/activity1/index.html (hunk 2 above)
Comment (shaikram) [reply shaikram_abdulguro/activity1/index.html:10]: yes sir, ginawa ko lang direct para mabilis ma pasa that time

**Pull request merged**